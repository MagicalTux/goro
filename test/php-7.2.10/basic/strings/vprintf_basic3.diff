     *** Testing vprintf() : basic functionality - using float format ***
002- 11.110000
003- int(9)
004- 11.110000
005- int(9)
006- 11.110000 22.220000
007- int(19)
008- 11.110000 22.220000
009- int(19)
010- 11.110000 22.220000 33.330000
011- int(29)
012- 11.110000 22.220000 33.330000
013- int(29)
002+ 11.11000000000000
003+ int(17)
004+ 11.11000000000000
005+ int(17)
006+ 11.11000000000000 22.22000000000000
007+ int(35)
008+ 11.11000000000000 22.22000000000000
009+ int(35)
010+ 11.11000000000000 22.22000000000000 33.33000000000000
011+ int(53)
012+ 11.11000000000000 22.22000000000000 33.33000000000000
013+ int(53)
     ===DONE===
