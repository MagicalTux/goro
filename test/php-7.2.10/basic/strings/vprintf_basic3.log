
---- EXPECTED OUTPUT
*** Testing vprintf() : basic functionality - using float format ***
11.110000
int(9)
11.110000
int(9)
11.110000 22.220000
int(19)
11.110000 22.220000
int(19)
11.110000 22.220000 33.330000
int(29)
11.110000 22.220000 33.330000
int(29)
===DONE===
---- ACTUAL OUTPUT
*** Testing vprintf() : basic functionality - using float format ***
11.11000000000000
int(17)
11.11000000000000
int(17)
11.11000000000000 22.22000000000000
int(35)
11.11000000000000 22.22000000000000
int(35)
11.11000000000000 22.22000000000000 33.33000000000000
int(53)
11.11000000000000 22.22000000000000 33.33000000000000
int(53)
===DONE===
---- FAILED
