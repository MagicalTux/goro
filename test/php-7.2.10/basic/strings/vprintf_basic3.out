*** Testing vprintf() : basic functionality - using float format ***
11.11000000000000
int(17)
11.11000000000000
int(17)
11.11000000000000 22.22000000000000
int(35)
11.11000000000000 22.22000000000000
int(35)
11.11000000000000 22.22000000000000 33.33000000000000
int(53)
11.11000000000000 22.22000000000000 33.33000000000000
int(53)
===DONE===